src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -39,6 +41,13 @@ protected override void DisconnectHandler(UIButton platformView)
 			base.DisconnectHandler(platformView);
 		}
 
+		public override void PlatformArrange(Rect rect)
+		{
+			base.PlatformArrange(rect);
+
+			UpdateValue(nameof(IImage.Source));
```

Comment:
This feels very expensive for iOS. On a re-layout, we reload the image and then resize it. If the layout changes slightly, this will keep happening. Not sure how bad this is, but for things like stream images, the stream may no longer be available and it is also not cached.

I wonder for this if it should rather _just_ trigger rescale/resize of the image instead of a full reload?